simulation:
  home:
    - [0.0, 0.0, 0.0]
  drop:
    - [-2.5, -1.5, 0.0]
  patrol:
    - [0.5, 0.0, 0.0]
    - [0.5, -1.5, 0.0]
    - [-1.0, -1.5, 0.0]
    - [-2.5, -1.5, 0.0]
    - [-1.0, -1.5, 0.0]
    - [0.5, -1.5, 0.0] # center
    - [2.0, -1.5, 0.0]
    - [3.5, -1.5, 0.0]
    - [2.0, -1.5, 0.0]
    - [0.5, -1.5, 0.0]
    - [0.5, 0.0, 0.0]
  home_to_drop:
    - [0.5, 0.0, 0.0]
    - [0.5, -1.5, 0.0]
    - [-1.0, -1.5, 0.0]
    - [-2.5, -1.5, 0.0]
  drop_to_home:
    - [-1.0, -1.5, 0.0]
    - [0.5, -1.5, 0.0]
    - [0.5, 0.0, 0.0]
    - [0.0, 0.0, 0.0]

real:
  home:
    - [1.5, 0.0, 0.0]
  drop:
    - [-1.5, 0.0, 0.0]
  patrol:
    - [1.5, 0.0, 0.0]
    - [3.0, 0.0, 0.0]
    - [4.5, -1.0, 0.0]
    - [5.5, 0.0, 0.0]
    - [4.5, -1.0, 0.0]
    - [3.0, 0.0, 0.0]
  home_to_drop:
    - [1.0, 0.0, 0.0]
    - [0.3, -0.80, 0.0]
    - [-1.0, -0.75, 0.0]
    - [-1.5, 0.0, 0.0]
  drop_to_home:
    - [-1.5, 0.0, 0.0]
    - [-1.0, -0.75, 0.0]
    - [0.3, -0.80, 0.0]
    - [1.0, 0.0, 0.0]
